coralproject/talk
# Removed 'rejected' and 'approved' comments from flagged queue.

## What does this PR do?

The flagged comments queue was showing all flagged comments, including those that had already been accepted or rejected. This PR removes those comments from the queue.

## How do I test this PR?

1. Turn Premod off.
2. Create a comment.
3. Flag it.
4. The flagged comment should appear in the admin panel under "flagged."
5. Accept or reject it.
6. It should disappear from the queue.
